packages/constants/src/constants.ts

```diff
@@ -131,3 +131,17 @@ export const MC_API_PROXY_TARGETS = {
   MC_METRICS: 'mc-metrics',
 } as const;
 export type TMcApiProxyTargets = (typeof MC_API_PROXY_TARGETS)[keyof typeof MC_API_PROXY_TARGETS];
+
+// Global application environment on window object
+export interface ApplicationWindow extends Window {
```

Comment:
Oh neato.

Reply:
Yeah I think it's better to ship this with the packages. I chose to put it in the constants package because it's usually used by all other packages.